package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@internetarchive/metadata-service",
-  "version": "0.1.0",
+  "version": "0.0.0-bc6513-0",
```

Comment:
Let's revert this version change before merging this in. Our approach is generally to update package versions in a standalone PR after merging one or more feature branches, alongside a publish to npm.

Reply:
Oh dear, I tried and yet failed to keep that temp version from getting pushed. Will remove it before merging!